homeassistant/components/demo/media_player.py

```diff
@@ -427,12 +422,11 @@ class DemoTVShowPlayer(AbstractDemoPlayer):
 
     # We only implement the methods that we support
 
-    _attr_device_class = MediaPlayerDeviceClass.TV
     _attr_media_content_type = MediaType.TVSHOW
 
     def __init__(self) -> None:
         """Initialize the demo device."""
-        super().__init__("Lounge room")
+        super().__init__("Lounge room", MediaPlayerDeviceClass.TV)
```

Comment:
This is actually a bugfix for `DemoTVShowPlayer` as the class attribute was being ignored.